lib/currency-locales/locales/da.yml

```
@@ -1,23 +1,59 @@
+---
 da:
   number:
     currency:
       format:
         delimiter: "."
-        format: "%n %u"
+        format: "%u %n"
         precision: 2
         separator: ","
         significant: false
         strip_insignificant_zeros: false
-        unit: "kr"
+        unit: DKK
```

New unit

---

this is wrong. this is the currency code not the symbol.

---

TwitterCldr::Shared::Currencies.for_code("DKK")
{currency: :DKK, name: "Danish Krone", cldr_symbol: "kr", symbol: "kr", code_points: [107, 114]}

```suggestion
        unit: kr
```